isthmus/src/main/java/io/substrait/isthmus/SqlToSubstrait.java

```diff
@@ -1,18 +1,37 @@
 package io.substrait.isthmus;
 
+import com.github.bsideup.jabel.Desugar;
 import com.google.common.annotations.VisibleForTesting;
 import io.substrait.extension.ExtensionCollector;
+import io.substrait.extension.SimpleExtension;
+import io.substrait.isthmus.expression.RexExpressionConverter;
+import io.substrait.isthmus.expression.ScalarFunctionConverter;
+import io.substrait.proto.Expression;
+import io.substrait.proto.Expression.ScalarFunction;
+import io.substrait.proto.ExpressionReference;
+import io.substrait.proto.ExtendedExpression;
+import io.substrait.proto.FunctionArgument;
 import io.substrait.proto.Plan;
 import io.substrait.proto.PlanRel;
+import io.substrait.proto.SimpleExtensionDeclaration;
+import io.substrait.proto.SimpleExtensionURI;
 import io.substrait.relation.RelProtoConverter;
 import io.substrait.type.NamedStruct;
-import java.util.List;
+import io.substrait.type.TypeCreator;
+import io.substrait.type.proto.TypeProtoConverter;
+import java.io.IOException;
+import java.util.*;
```

Comment:
same here.

Reply:
This case, I defer to `./gradlew spotlessApply`

Reply:
spotless doesn't actually squash imports down like this. What IDE are you using? I know with IntelliJ it defaults to doing this when you autoimport. There's a way to turn it [off](https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports)